README.Rmd

```diff
@@ -132,6 +133,14 @@ rmd$add_md(knitr::kable(new_table), where = 2L)
 rmd$head(21)
 ```
 
+## Background / pre-requisites
```

Comment:
I think this needs to be linked on line 26 so that people who are completely lost will know where to go.